pages/02.install/06.truenas-scale/default.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'TrueNAS Scale Install'
+---
+
+### Before you start
+In your server's filesystem, create a folder for your library and a folder for Kavita's config.
+
+### Installing Kavita (Stable)
+
+[Install](https://truecharts.org/manual/Quick-Start%20Guides/02-Installing-an-App/#requirements=) the TrueCharts app catalog, search for Kavita, and click "install".
+
+Modify the following sections:
+- Application Name
+  - Set the application name to whatever you want.
+- Storage and Persistence
+  - Set the storage type for "App Config Storage" to "HostPath (Simple)" and set the path to your config folder.
+  - Set the storage type for "Manga Storage" to "HostPath (Simple)", and set the path to your library folder. This will mount to `/manga` within Kavita.
+
+![Kavita storage settings](storage.png)
+
+Leave all other sections at their default settings. Continue to the end and click "Save". 
+
+To access Kavita, go to the Installed Applications page and click Kavita's "Open" button.
+
+### Installing Kavita (Nightly)
+
+As there currently isn't a TrueCharts app for Kavita's nightly releases, it needs to be installed as a Docker image.
+
+Go to the Available Applications page and click "Launch Docker Image".
+
+Modify the following sections:
+- Application Name
+  - Set the application name to whatever you want.
+- Container Images
+  - Set the image repository to `kizaing/kavita`.
+  - Set the image tag to `nightly`.
```

Comment:
This should be `latest`. We don't want users to be on nightly by default, as that's for beta testers and can be unstable.

Reply:
The only reason to chose the "Launch Docker Image" method over the official TrueCharts app is to use the nightly release, hence why I made it the default tag and called the section "Kavita (Nightly)".

If you are against users using `nightly` by default, perhaps I could add a disclaimer that says unless they want to test the latest changes, users should stick to the TrueCharts method. How does this sound?

Reply:
Ahh...I missed the two different sections. Yeah, I would put an additional disclaimer underneath to inform the users this is a version for beta testers and there may be bugs. Maybe linking to the discord dev-branch-testing (https://discord.com/channels/821879810934439936/851515194185744464) would be a good idea as well.